ace-link-dashboard.el

```diff
@@ -54,7 +55,7 @@
   (interactive)
   (let ((point (avy-with 'ace-link-dashboard-remove
                  (avy-process
-                  (mapcar #'cdr (ace-link-dashboard--collect))
+                  (mapcar #'cdr (ace-link-dashboard--collect 'in-section))
```

Comment:
This could be another function `ace-link-dashboard--remove-collect` that uses `ace-link-dashbord--collect` and then filter that list based on some criteria, maybe the `face` property.